sdk/iOS/OoyalaSkinSDK/OOSkinPlayerObserver.m

```diff
@@ -347,6 +347,12 @@ - (void)bridgeCurrentItemChangedNotification:(NSNotification *)notification {
 - (void)bridgeStateChangedNotification:(NSNotification *)notification {
   NSString *stateString = [OOOoyalaPlayerStateConverter playerStateToString:self.player.state];
   
+  NSNumber *stateObject = notification.userInfo[@"newState"];
+  if (stateObject) {
```

Comment:
I think this should be default behavior, i.e.:
```suggestion
NSNumber *stateObject = notification.userInfo[@"newState"] ?: self.player.state;
NSString *stateString = [OOOoyalaPlayerStateConverter playerStateToString:stateObject];
```

Reply:
Agree, if it is okey that if notification.userInfo[@"newState"]  is nil (in unknown reason) observer will get string for initial state in enum: OOOoyalaPlayerStateInit

Reply:
Can't do it in this way.
self.player.state - it's NSUInteger, but  notification.userInfo[@"newState"] it's NSNumber.
But anyway I simplified this snippet.

Reply:
@maxkupetskii  Please, review it again

Reply:
Use @(self.player.state) in my snippet

Reply:
Done